### PR TITLE
Add command logging

### PR DESCRIPTION
**Pull request by tnederlof**
Closes https://github.com/sol-eng/wbi/issues/157

This PR adds a secondary logger in an internal logger package which is created when the CLI starts and records the commands taken to install/configure. The output is a ready to go bash script with a `.sh` extension, which can be run on an identical server. This opens the door for future work in HA and other multi-server setups.

### CODE REVIEW

**Review by samcofer — COMMENTED** [review]
Looks pretty good. Works well on the server, just a couple flow notes

**Comment by samcofer** [comment]
@tnederlof Thoughts on my feedback?

File: internal/logging/command.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	cmdlog.Warnf(format, v...)
+}
+
+// Error ...
+func Error(format string, v ...interface{}) {
+	cmdlog.Errorf(format, v...)
+}
+
+// Detect which operating system WBI is running on
+// this is duplicated in operatingsystem/detect.go but cannot import that package here
```
[the remark this answers is not in the document]
Comment (tnederlof) [reply internal/logging/command.go:67]: yes GO has issues with circular dependencies so I was forced to move it here/recreate it. There may be a more elegant way to do this but I tried for a while and failed.

File: internal/workbench/install.go [hunk 2]
```diff
@@ -77,6 +78,13 @@ func DownloadAndInstallWorkbench(osType config.OperatingSystem) error {
 	if err != nil {
 		return fmt.Errorf("InstallWorkbench: %w", err)
 	}
+	// save to command log
+	installCommand, err := RetrieveInstallCommandForWorkbench(installerInfo.BaseName, osType)
+	if err != nil {
+		return fmt.Errorf("RetrieveInstallCommand: %w", err)
+	}
+	cmdlog.Info("curl -O " + installerInfo.URL + "\n")
```
[the remark this answers is not in the document]
Comment (tnederlof) [reply internal/workbench/install.go:86]: Good idea, just implemented it in latest commit

File: internal/logging/command.go [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	f, err := os.OpenFile(logFile, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0700)
+	if err != nil {
+		log.Fatalf("error opening file: %v", err)
+	}
+
+	cmdlog = logrus.New()
+
+	cmdlog.SetOutput(f)
+	cmdlog.SetFormatter(new(MyFormatter))
+
```
[the remark this answers is not in the document]
Comment (samcofer) [reply internal/logging/command.go:40]: Yup, let's ignore this suggestion for now.

**Review by samcofer — APPROVED** [review]
Looks good to me!

**Pull request merged**